iothub/service/src/Exceptions/IotHubServiceException.cs

```diff
@@ -109,10 +109,12 @@ private static bool DetermineIfTransient(HttpStatusCode statusCode, IotHubServic
                 IotHubServiceErrorCode.IotHubQuotaExceeded
                     or IotHubServiceErrorCode.DeviceNotOnline
                     or IotHubServiceErrorCode.ThrottlingException
+                    or IotHubServiceErrorCode.ThrottlingBacklogTimeout
                     or IotHubServiceErrorCode.ServerError
                     or IotHubServiceErrorCode.ServiceUnavailable
                     => true,
-                IotHubServiceErrorCode.Unknown => statusCode == HttpStatusCode.RequestTimeout,
+                IotHubServiceErrorCode.Unknown => statusCode == HttpStatusCode.RequestTimeout
+                    || statusCode == (HttpStatusCode)429,
```

Comment:
Isn't this getting mapped to "IotHubServiceErrorCode.ThrottlingException"? If it isn't already then should we?

Reply:
It might be, but it occurred to me we wouldn't have had this issue if we had this line.

Reply:
Sorry 429 is not associated. We parse the error code (string, not the numeric status code). That is 429001.